src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/SyntaxTreeIndex_Create.cs

```diff
@@ -381,5 +394,167 @@ private static void Free(bool ignoreCase, HashSet<string> identifiers, HashSet<s
             SharedPools.StringHashSet.ClearAndFree(identifiers);
             SharedPools.StringHashSet.ClearAndFree(escapedIdentifiers);
         }
+
+        private static void ProcessStringLiteralForGlobalAttributeInfo(
+            SyntaxToken stringLiteral,
+            ISyntaxFacts syntaxFacts,
+            PooledDictionary<string, ArrayBuilder<int>> globalAttributeInfoBuilder)
+        {
+            var text = stringLiteral.ValueText;
+            if (string.IsNullOrEmpty(text) ||
+                !IsTargetOfGlobalAttribute(stringLiteral, syntaxFacts))
```

Comment:
can we also just limite this to attributes syntactically called SuppressWhatever?